utils/api/volume.js

```diff
@@ -15,12 +13,19 @@ const getETHPosition = pair => {
   }
 };
 
+/**
+ *
+ * @param connex the connex provider object
+ * @param web3 the web3 provider object
+ * @param address address of the pair contract
+ * @returns {Promise<number>} done volume in a 24H period, denominated in VET
+ */
 const getVolume = async(connex, web3, address) => {
-	const dater = new EthDater(web3);
 
-	const from = moment().subtract(1, 'days').format()
-	const { block: FROM_BLOCK } = await dater.getDate(from, true);
 	const { number: TO_BLOCK } = connex.thor.status.head;
+	// As vechain blocks are always 10s,
```

Comment:
Can vechain miss blocks? So could this be >= 24 hours?

Reply:
nope, they're proof of authority so I don't think they miss blocks. Also I did a random check while debugging yesterday, and the `to` and `from` blocks are 24 hours apart

Reply:
Hmm, if a node misses its slot i would've thought they can miss a block, that's how most PoA / PoS systems work. The diff shouldn't be huge though so im not too worried